dnsdist-1.9.yaml

```diff
@@ -0,0 +1,199 @@
+package:
+  name: dnsdist-1.9
+  version: "1.9.10"
+  epoch: 0
+  description: PowerDNS dnsdist — highly DNS-, DoS- and abuse-aware load-balancer
+  copyright:
+    - license: GPL-2.0-only
+  dependencies:
+    provides:
+      - dnsdist=${{package.full-version}}
+  scriptlets:
+    pre-install: |
+      #!/bin/sh
+      addgroup -S dnsdist 2>/dev/null
+      adduser  -S -D -H -h /var/empty -s /bin/false -G dnsdist -g dnsdist dnsdist 2>/dev/null
+      exit 0
+
+vars:
+  py-version: 3.12 # Pinning 3.12 due to imghdr. See: https://peps.python.org/pep-0594/#imghdr
+  lua-version: 5.4 # Pinning latest lua version to pass cmake
+
+environment:
+  contents:
+    packages:
+      - boost-dev
+      - build-base
+      - busybox
+      - cargo-auditable
+      - cmake
+      - fstrm-dev
+      - gnutls-dev
+      - libbpf-dev # eBPF/XDP support
+      - libcap-dev
+      - libedit-dev
+      - libquiche
+      - libsodium-dev
+      - libtool
+      - lmdb-dev
+      - lua${{vars.lua-version}}-dev
+      - luajit-dev
+      - mtdev-dev
+      - net-snmp-dev
+      - nghttp2-dev
+      - openssl-dev
+      - pkgconf-dev
+      - protobuf-dev
+      - py3-yaml
+      - python-3.12
```

Comment:
Am not sure how to reference lines but in here https://eng.inky.wtf/docs/packages/reference/python/#development-best-practices, the suggestion is to use `py3.x-build-base` and since you already created the py-version var, we can make use of that here

Reply:
Nice idea, it makes sense! https://github.com/wolfi-dev/os/pull/61885/commits/953364aadbac7e76d2e23e00167cd6968e58150a